File: app/src/main/java/com/a60n1/ejashkojme/models/Ride.java
package com.a60n1.ejashkojme.models;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Ride {

    public String from;
    private String address;

    public Ride() {

    }

    public Ride(String from, String address) {
        this.from = from;
        this.address = address;
    }
}

File: app/src/main/java/com/a60n1/ejashkojme/models/Ride.kt
package com.a60n1.ejashkojme.models

import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
class Ride {
    var from: String? = null
    private var address: String? = null

    @Suppress("unused")
    constructor()

    constructor(from: String?, address: String?) {
        this.from = from
        this.address = address
    }
}